readme.md

```diff
@@ -28,35 +28,19 @@ Please make sure you install the following tools before starting with the instal
 
 ## Installation
 
-> Note that you're free to adjust the `~/Sites/pastebin` location to any directory you want on your machine.
+> Note that you're free to adjust the `~/Code/pastebin` location that will be in the `Homestead.yaml` to any directory you want on your machine.
 
 1. Clone this repository: `git clone git@github.com:laravelio/pastebin.git pastebin`
-2. Add the `Homestead.yaml` file from below to the root of your project
+2. Run `./vendor/bin/homestead make`
```

Comment:
I've read the help details for this command but from my understanding it needs the `--example` flag in order to copy the `Homestead.yaml.example` file, correct? What will this command do besides that?